github-webhook-trigger/internal/webhook/webhook.go

```diff
@@ -0,0 +1,188 @@
+package webhook
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/go-playground/webhooks/v6/github"
+	"github.com/google/uuid"
+	"github.com/konstellation-io/kai-sdk/go-sdk/sdk"
+	"github.com/konstellation-io/kai-sdk/go-sdk/sdk/messaging"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+const (
+	path = "/webhooks"
```

Comment:
As we will be listening to a single webhook, I'd change the path to singular.

```suggestion
	path = "/webhook"
```